plugin.go

```diff
@@ -3,14 +3,23 @@ package web
 
 import (
 	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/launchrctl/launchr/pkg/log"
 
 	"github.com/launchrctl/launchr"
 	"github.com/launchrctl/web/server"
 	"github.com/spf13/cobra"
 )
 
-// APIPrefix is a default api prefix on the server.
-const APIPrefix = "/api"
+const (
+	// APIPrefix is a default api prefix on the server.
+	APIPrefix             = "/api"
+	stopArg               = "stop"
+	webPidFile            = "/tmp/launchr-web.pid"
+	cobraBackgroundEnvVar = "BACKGROUND"
```

Comment:
let's use opposite: have background by default

+ have option to keep foreground

Reply:
Let's add option to keep the execution foreground (we will need it if web ui ever runs in a container exposed to platform users for instance)